web-js/register-server-token-with-identity-manager/README.md

```diff
@@ -0,0 +1,63 @@
+#Register Server Token with Identity Manager
+
+##About
+The IdentityManager class manages user credentials for secure resources. After credentials have been registered with the IdentityManager tokens will be automatically appended to requests to secured resources. In some cases developers may want to generate a token using custom code and later register the token with the IdentityManager. This allows the developer to control how a token is created but does not force them to control how the token is used. oAuth tokens can easily be registered with IdentityManager using it's registerToken method. However ArcGIS Server does not issue oAuth tokens. Registering an ArcGIS Server token with IdentityManager is more complex.
```

Comment:
> This allows the developer to control how a token is created.

you either need to make it clearer in the README or in the app itself *why* anyone would find it beneficial to get this control?  i'm pretty sure the reason would be designing a login mechanism outside/prior to the map view, but this should be totally explicit because its the crux of the whole demo.